week-1/Homework/mandatory/3-function-output.js

```diff
@@ -1,16 +1,18 @@
 // Add comments to explain what this function does. You're meant to use Google!
-function getNumber() {
-    return Math.random() * 10;
+
+function getNumber() { 
+    return Math.random() * 10;// this funtion returns a random number then multiplys by 10.
```

Comment:
Need more explanation here to show you understand. Use following as a guide:
- Math.random returns a randoms a number between ..... and .....
- Then the result of Math.random() is multiplied by 10 so that the number returned from the function is between .... and ....